build.sh

```diff
@@ -208,6 +208,19 @@ isMSBuildOnNETCoreSupported()
         elif [ "$__BuildOS" == "OSX" ]; then
             __isMSBuildOnNETCoreSupported=1
         fi
+    elif [ "$__BuildArch" == "arm" ] || [ "$__BuildArch" == "arm64" ] ; then
+        if [ "$__BuildOS" == "Linux" ]; then
+            if [ "$__DistroName" == "ubuntu" ]; then
+                __isMSBuildOnNETCoreSupported=1
```

Comment:
I would certainly remove OS X and probably RHEL, Debian is likely going to be similar to Ubuntu but it would probably be easier to decide on one target initially.